README.md

```diff
@@ -9,87 +9,105 @@
 [![support](https://img.shields.io/badge/sponsor-Open%20Collective-blue.svg)](https://opencollective.com/tauri)
 
 
-## Usage
+# Usage
 
-With Bash:
+## Bash:
 
-``` bash
+### curl
+``` 
 sh <(curl https://create.tauri.app/sh)
-# or
+```
+### wget
+```
 sh <(wget https://create.tauri.app/sh)
 ```
 
-With Powershell:
+## Powershell:
 
 ``` powershell
 iwr -useb https://create.tauri.app/ps | iex
 ```
 
-With Cargo:
+## Cargo:
 
 ``` bash
 cargo install create-tauri-app
 cargo create-tauri-app
 ```
 
-With NPM:
+## NPM:
 
 ``` bash
 npm create tauri-app@latest
 ```
 
-With Yarn:
+## Yarn:
 
 ``` bash
 yarn create tauri-app
 ```
 
-With PNPM:
+## PNPM:
 
 ``` bash
 pnpm create tauri-app
 ```
 
-You can also directly specify the project name and the template you want to use via additional command line options. For example, to scaffold a Vite + Vue project, run:
+<br>
 
-```bash
-# curl
+ Additional command line options allow you to directly specify the template to use.
+---
+
+### curl
+```
 sh <(curl https://create.tauri.app/sh) --template svelte
-# wget
+```
+### wget
+```
 sh <(wget https://create.tauri.app/sh) --template svelte
-# cargo
+```
+### cargo
+```
 cargo create-tauri-app --template svelte
-# npm 6.x
+```
+### npm 6.x
+```
 npm create tauri-app@latest my-tauri-app --template svelte
-# npm 7+, extra double-dash is needed:
+```
+### npm 7+, extra double-dash is needed:
+```
 npm create tauri-app@latest my-tauri-app -- --template svelte
-# yarn
+```
+### yarn
+```
 yarn create tauri-app my-tauri-app --template svelte
-# pnpm
+```
+### pnpm
+```
 pnpm create tauri-app my-tauri-app --template svelte
 ```
 
 Currently supported template presets include:
 
-- `vanilla`
-- `vanilla-ts`
-- `vue`
-- `vue-ts`
-- `react`
+- `angular`
+- `clojurescript`
+- `next-ts`
+- `next`
+- `preact-ts`
+- `preact`
 - `react-ts`
-- `svelte`
-- `svelte-ts`
-- `solid`
+- `react`
 - `solid-ts`
+- `solid`
+- `svelte-ts`
+- `svelte`
+- `vanilla-ts`
+- `vanilla`
+- `vue-ts`
+- `vue`
 - `yew`
```

Comment:
Keep the list of templates ordered as they appear in the CLI prompt list.

Reply:
Why is the CLI prompt list in this specific order?

Reply:
new templates are added to the bottom